examples/classification/classification/workflow.py

```diff
@@ -65,24 +78,83 @@ class Inference(BaseInference):
 )
 
 
+@dataclass(frozen=True)
+class TestSampleMetricsSingleClass(MetricsTestSample):
+    """Image-level metrics for Classification workflow."""
+
+    is_correct: bool
+    """An indication of the `classification_label` matching the associated ground truth label."""
+
+    is_TP: bool
+    """
+    An indication of the `classification_label` correctly matching the associated positive ground
+    truth label.
+    """
+
+    is_FP: bool
+    """
+    An indication of the `classification_label` incorrectly matching the associated negative ground
+    truth label.
+    """
+
+    is_FN: bool
+    """
+    An indication of the `classification_label` incorrectly matching the associated positive ground
+    truth label.
+    """
+
+    is_TN: bool
+    """
+    An indication of the `classification_label` correctly matching the associated negative ground
+    truth label.
+    """
+
+    classification: ScoredClassificationLabel
+    """
+    The model classification for this image. Empty when no inferences have a sufficient confidence score determined
+    by the `ThresholdConfiguration`.
+
+    For binary classification, if only one label's confidence score is provided as part of `Inference`, then
+    `classification` will reflect that.
+    """
+
+    threshold: float
+    """
+    The threshold used in evaluation — specified by the `ThresholdConfiguration`. It is commonly used in a binary
+    classification problem.
+    """
+
+    missing_inference: bool = False
+
+    def __post_init__(self):
+        object.__setattr__(self, "missing_inference", self.classification is None)
+
+
 @dataclass(frozen=True)
 class TestSampleMetrics(MetricsTestSample):
```

Comment:
Took me some time to figure out what's the difference between `TestSampleMetricsSingleClass` and `TestSampleMetrics` (hopefully I'm not wrong). They have the same class description. Is it intentional to have them the same? Would it be helpful to describe which class to use for which problem?

Reply:
Thanks for catching it.  No they are not supposed to be the same. I will update it to be more descriptive.